addons/account_edi_ubl_cii/models/res_partner.py

```diff
@@ -8,6 +8,15 @@
 from odoo.addons.account.models.company import PEPPOL_DEFAULT_COUNTRIES
 
 
+SANITIZE_PEPPOL_ENDPOINT_RE = re.compile(r'[^a-zA-Z\d\-._~]')
+
+
+def sanitize_peppol_endpoint(peppol_endpoint):
+    if not peppol_endpoint:
+        return peppol_endpoint
+    return SANITIZE_PEPPOL_ENDPOINT_RE.sub('', peppol_endpoint or '')
```

Comment:
Why `peppol_endpoint or ''` since you check before that it's Truthy ? :thinking:

Reply:
10 version later..., didn't cherry-pick the last one, ..loosing faith in myself :weary:, fix btw thanks for having spotted this

Reply:
Haha, no big deal, happens to the best of us :grin: